frontend-nextjs/components/layouts/readme.md

```
@@ -0,0 +1 @@
+共通して利用するレイアウト関連のコンポーネント(Header, Footerなど、なくても可
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)